entrypoint.sh

```diff
@@ -437,9 +437,13 @@ get_environment () {
         error "Environment not found: $environment"
     fi
 
+    # We can't use the environment name, we need to use the ID it maps to, so
+    # do this magic to get that.
+    id=$(rancher env | grep $environment | awk '{ print $1 }')
+
```

Comment:
maybe we should add an `if` block in here in case it fails to find the id?

and then follow that with a successful debug statement, like:

```
debug "Found id: $id for environment: $environment"
```

Reply:
Updated